src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -362,9 +367,10 @@ else if (TokenTypes.isCommentType(aToken)) {
      * @param aAST the root AST
      * @param aContents the contents of the file the AST was generated from.
      * @param aAstState state of AST.
+     * @throws NullPointerException if NullPointerException was met.
      */
     private void walk(DetailAST aAST, FileContents aContents
-            , AstState aAstState)
+            , AstState aAstState) throws NullPointerException
```

Comment:
`throws NullPointerException` doesn't make sense to me - every method in Java can potentially throw NPE